islandora_datastream_crud.drush.inc

```diff
@@ -510,20 +516,34 @@ function drush_islandora_datastream_crud_push_datastreams() {
  */
 function drush_islandora_datastream_crud_delete_datastreams() {
   $dsid = drush_get_option('dsid');
+  $versions = drush_get_option('versions');
   if ($dsid == 'DC') {
     drush_user_abort(dt('DC is a required datastream and cannot be deleted.'));
     exit;
   }
-  if (!drush_confirm(dt("You are about to delete (purge) the !dsid " .
-    "datastream from a set of objects in your repository. This action " .
-    "cannot be undone. Do you want to want to continue?",
-    array('!dsid' => drush_get_option('dsid'))))) {
+  module_load_include('inc', 'islandora_datastream_crud', 'includes/utilities');
+  if($versions) {
```

Comment:
If you set `--versions=0`, which should mean "The first version of the datastream only", this code reads `$versions` as null.

I tried printing some text under the "else" at line 533, and when running with `--versions=0`, the text printed.

Reply:
I tried changing the IF to `if ($versions >= 0)`, which allows you to purge only version 0, but it also triggers if you don't include `--versions` at all. So that's problematic.

Reply:
This one works: `if (!is_null($versions))`